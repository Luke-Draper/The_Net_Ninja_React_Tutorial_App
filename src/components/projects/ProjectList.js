import React from "react";
import ProjectSummary from "./ProjectSummary";

const ProjectList = () => {
	return (
		<div className="section ProjectList">
			<ProjectSummary />
			<ProjectSummary />
			<ProjectSummary />
			<ProjectSummary />
			<ProjectSummary />
		</div>
	);
};

export default ProjectList;
