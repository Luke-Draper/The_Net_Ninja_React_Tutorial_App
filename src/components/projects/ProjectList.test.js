import React from "react";
import ProjectList from "./ProjectList";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("ProjectList renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<ProjectList />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
