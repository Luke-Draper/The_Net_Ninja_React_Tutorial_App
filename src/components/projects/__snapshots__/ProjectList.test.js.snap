// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectList renders without crashing 1`] = `
<div
  className="project-list"
>
  <h3>
    ProjectList
  </h3>
</div>
`;
